packages/experimental/Tooltip/src/Tooltip.tsx

```diff
@@ -0,0 +1,41 @@
+/** @jsxRuntime classic */
+/** @jsx withSlots */
+import * as React from 'react';
+
+import { ensureNativeComponent } from '@fluentui-react-native/component-cache';
+import { compose, mergeProps, withSlots } from '@fluentui-react-native/framework';
+import type { UseSlots } from '@fluentui-react-native/framework';
+
+import type { TooltipType, TooltipProps } from './Tooltip.types';
+import { tooltip } from './Tooltip.types';
+import { useTooltip } from './useTooltip';
+
+const NativeTooltipView = ensureNativeComponent('RCTTooltip');
+
+/**
+ * A function which determines if a set of styles should be applied to the component given the current state and props of the tooltip.
+ *
+ * @param layer The name of the state that is being checked for
+ * @param userProps The props that were passed into the tooltip
+ * @returns Whether the styles that are assigned to the layer should be applied to the tooltip
+ */
+export const tooltipLookup = (layer: string, userProps: TooltipProps): boolean => {
+  return userProps[layer];
+};
+
+export const Tooltip = compose<TooltipType>({
```

Comment:
>compose [](http://example.com/codeflow?start=23&length=7)

Seems like this is meant to be a wrapper around a native component. In that case, compose might be too much if styling isn't going to be driven by JS. If this component is meant to be a wrapper around a native component it might make more sense to use stagedComponent instead which is a lighter framework.

Reply:
Thanks for the suggestion! Updated to using stagedComponent.